lib/widgets/speaker_detail/speaker_detail_header.dart

```diff
@@ -0,0 +1,104 @@
+import 'package:flutter/material.dart';
+import 'package:hackathon_flutterturkiye_org/core/styles/theme_helper.dart';
+import 'package:hackathon_flutterturkiye_org/core/utils/responsive_helper.dart';
+import 'package:hackathon_flutterturkiye_org/widgets/speaker_detail/speaker_detail.dart';
+
+class SpeakerDetailHeader extends StatelessWidget {
+  const SpeakerDetailHeader({
+    Key key,
+    @required this.speakerImage,
+    @required this.speakerName,
```

Comment:
Missing assertions.

Reply:
My bad. Probably, I forgot it. I will handle it.